aait/web/group-3/starter_project_group3/components/header/Header.tsx

```diff
@@ -64,7 +64,7 @@ const Header: React.FC = () => {
           {!isLoggedIn && (
```

Comment:
You can wrap the two Links inside the same logic like: 
`
<> 
{!isLoggedIn && (
            <Link
              href={"/login"}
              className="text-login_color hidden nav_bar_screen:inline-block p-3 border-2 border-white rounded-lg hover:border-primary transition ease-in-out duration-200"
            >
              Login
            </Link>
<Link
              href={"/register"}
              className="p-3 bg-primary text-white rounded-lg hidden nav_bar_screen:inline-block border-2 border-primary hover:bg-white hover:text-primary transition ease-in-out duration-200"
            >
              Register
            </Link>
</>
`